examples/string-functions/string-functions.go

```diff
@@ -1,43 +1,43 @@
-// The standard library's `strings` package provides many
-// useful string-related functions. Here are some examples
-// to give you a sense of the package.
+// Il pacchetto `string` della standard library contiene
```

Comment:
*strings, in questo caso è proprio il nome del pachetto che si chiama strings, chiamarlo string è errato perché non puoi fare `import "string"`